port_ocean/core/utils.py

```diff
@@ -29,6 +30,17 @@ def is_same_entity(first_entity: Entity, second_entity: Entity) -> bool:
     )
 
 
+async def validate_integration_runtime(
+    port_client: PortClient, requested_runtime: Runtime
+):
+    current_integration = await port_client.get_current_integration()
+    current_runtime = current_integration.get("installationType", "OnPrem")
+    if current_integration and current_runtime != requested_runtime:
```

Comment:
Why wont u have the integration

Reply:
it doesn't necessarily exist, what if this is the first time running...